### PR TITLE
The Newsweek Page Clone

### PR DESCRIPTION
**Pull request by KinezaEnaelle**
This project is about cloning the Newsweek page using HTML5/CSS3 and Bootstrap4. In this project, we used to flex and grid bootstrap classes to position items on the page.

### CODE REVIEW

File: assets/styles/main.css [hunk 1]
```diff
@@ -0,0 +1,1699 @@
...
+  text-rendering: optimizeSpeed;
+  line-height: 1.5;
+}
+
+img {
+  max-width: 100%;
+  display: block;
+  object-fit: cover;
+}
+
```
Comment (jubaan) [inline assets/styles/main.css]: ### [Comment]
- Just for your information: Boostrap already uses a `reset stylesheet`

File: assets/styles/main.css [hunk 2]
```diff
@@ -0,0 +1,1699 @@
...
+/* HEADER SECTION */
+
+.navcontainer {
+  position: relative;
+  width: 1250px;
+  margin: 0 auto;
+}
```
Comment (jubaan) [inline assets/styles/main.css]: - [ ] Bootstrap has a class for  `container` and `container-fluid`

- [ ] Bootstrap has class for `position-relative`

File: assets/styles/main.css [hunk 3]
```diff
@@ -0,0 +1,1699 @@
...
+  align-content: center;
+}
+
+.middlenav {
+  padding: 20px;
+}
+
+.search {
+  border: 0 none;
+}
```
Comment (jubaan) [inline assets/styles/main.css]: - [ ] Bootstrap has `border-0`

File: assets/styles/main.css [hunk 4]
```diff
@@ -0,0 +1,1699 @@
...
+  height: 16px;
+}
+
+.nav-red-right span {
+  color: #fff;
+}
+
+.nav-red-right p {
+  color: #fff;
+}
```
Comment (jubaan) [inline assets/styles/main.css]: - [ ] Bootstrap has classes for white color.

**Review by jubaan — CHANGES_REQUESTED** [review]
# Status: Changes Required :recycle: 

## :wave: :wave: Hello @KinezaEnaelle  and @Davidosky007 . Thanks for submitting you project!

Great thing this project has and has done well:
:100: Great solid start!
:100: Nav, Main, and Footer contain all element needed
:100: Bootstrap is well implemented
:100: elements change in the different screen sizes

There are some issues that need to be addressed before we can move forward. Please review the list below.

## Required changes :wrench: 
### [Mandatory] :red_circle: 

- [x] The preview screenshot is not displaying in the README

- [x] Use more Bootstrap classes. see inline

- [x] subscribe and login are not aligned to the bottom
image

- [ ] Category element is not displaying in the correct position
image

- [ ] my turn Images are not displaying properly
image

- [x] More stories are not looking as original
image

- [x] subscribe element is not centered in the column
image

- [x] subscribe and newsletter section are not displaying properly
image

- [x] Footer is not displaying properly. Social icons out of place, margins/paddings are incorrect between elements and around the footer container
image

- [x] Navbar in tablet view is not showing menu
image

- [x] Signup element not displaying as original
image

- [x] Clone doesn't have margin left in tablet mode for some sections
image

- [x] Subscribe, Newsletter and footer are not displaying correctly in tablet view
image

- [x] login icon is mission in navbar
image

- [x] The order of the sections are not being followed in the mobile view
image

- [x] Unaligned elements in more stories
image

- [x] The debate section has unaligned elements, is not showing as original
image

- [ ] sign up is not displaying as original in mobile view
image

- [ ] In the magazine section is not displaying as original
image

- [ ] Elements in the editors pick section are not displaying as the original, there are no space between elements, elements are not well positioned, and images are not a similar size
image

- [ ] Elements in other section are not displaying correctly, elements are unaligned, the titles of the section doesn't have a similar style as the original
image

- [ ] Subscribe section is not looking as original
image

- [ ] Newsletter doesn't look as the original
image

- [ ] Footer doesn't look as the original
image

### [OPTIONAL]  :yellow_circle:  

- [ ] Use [semantic HTML](https://www.w3schools.com/html/html5_semantic_elements.asp) for `<nav>`, `<main>`, and `<article>`s.

### [Comments] :green_circle: 
- [Bootstrap documentation](https://getbootstrap.com/docs/4.0/getting-started/introduction/) Please go through the Bootstrap Documentation and search for the CSS styles. This will help reduce the CSS stylesheet file. I just mention some improvements but there are more properties that you can use from bootstrap.
## Keep up the great work :muscle: and Happy Coding :smiley: :computer:!!

File: assets/styles/main.css [hunk 5]
```diff
@@ -0,0 +1,326 @@
...
+.sponsor-header {
+  font-size: 20px;
+}
+
+.card {
+  background-color: #e2e0e0;
+}
+
+.news-signup {
+  background-color: #e2e0e0 !important;
```
Comment (arslanbisharat) [inline assets/styles/main.css]: - [ ] PLEASE REMOVE !IMPORTANT

File: README.md [hunk 6]
```diff
@@ -1,2 +1,80 @@
-# News-Week-Clone
-An Html , CSS Project Where we use Bootstrap for responsiveness.
+
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
```
Comment (arslanbisharat) [inline README.md]: Please remove this line

File: README.md [hunk 7]
```diff
@@ -1,2 +1,80 @@
...
+
+# # News-Week-Clone
```
Comment (arslanbisharat) [inline README.md]: Please remove one #

File: README.md [hunk 8]
```diff
@@ -1,2 +1,80 @@
...
+
+Feel free to check the [issues page](https://github.com/Davidosky007/News-Week-Clone/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
```
Comment (arslanbisharat) [inline README.md]: Whose code you used?

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested 💀 

Hello Team, 👋 

Thank you for making changes. I would suggest some other changes.
**Please have a look below**

- [ ] [OPTIONAL] Too many font-size declarations (16), abstraction needed.

- [ ] Navbar on tablet mode has hamburger but its position it not aligned correctly.. Please have a look at the original page

- [ ] Your whole webpage has extra spacing around the edges. Please have a look there and work on it. . 

- [ ] On tablet mode, newsletter email input has more width than the original one. Please work on it. 
- [ ] On small screen, the Newsletter section doesn't look like the original one. Please work on it. 
- [ ] Navbar on small mode has hamburger but its position it not aligned correctly.. Please have a look at the original page

--------------------------------------------------------------------------------------------------

**Regards,
Arslan Bisharat
slack: arslanbisharat**

File: index.html [hunk 9]
```diff
@@ -0,0 +1,2267 @@
...
+      <header>
+        <!--top navigation bar with logo-->
+        <nav
+          class="navbartop bg-color d-lg-flex d-none p-3 justify-content-around position-relative"
+        >
+          <div class="weather flex-fill">
+            <div class="headcont d-flex">
+              <img
+                src="./assets/images/weather.png"
+                alt=""
```
Comment (umairarshadbutt) [inline index.html]: - Add text in `alt` attribute
- Update it in all pages

**Review by umairarshadbutt — CHANGES_REQUESTED** [review]
# Status: REQUIRED CHANGES :warning: :construction: 

Hi 
Good job with the changes! :tada: We are almost there.
We, however, do have a few changes to carry out before we proceed (see review comment).

##

 For this project, we have to make sure nothing is messed up. Kindly compare the screenshots below 👇 and work on the suggested changes.
- [x] Please add the text in `alt` attribute. See the inline comment. ([Article](https://www.w3schools.com/tags/att_img_alt.asp) for better understanding)

### Newsletter
 - [x] Please design the **Newsletter** like original landing page.

|[Source Page](https://www.newsweek.com/)|[Clone Page](https://davidosky007.github.io/News-Week-Clone/)|
|:---:|:---:|
|source-page|clone|

##
- **When you have made the specified changes, please submit another code review. Thank you!**
##
**If you are stuck at any point or have any questions, let me know on slack at any time. I will be happy to help**
HAPPY CODING :computer:
**[Umair Arshad](https://umairarshad.me/)**
[Twitter](https://twitter.com/its_UmairArshad) | [LinkedIn](https://www.linkedin.com/in/umair-arshad-butt/) | [GitHub](https://github.com/umairarshadbutt)

**Review by arslanbisharat — APPROVED** [review]
# Approved :smile: 

Hello Developer, :wave:

I just reviewed your project and I am very happy to announce that your project is approved.
Congratulations on achieving another milestone in your career. :clap:

_I am looking forward to seeing your next project._

-----------------------------------------------------------------------------------------------

#### Regards,

#### Arslan Bisharat

#### slack: arslanbisharat

**Pull request merged**